docs/http/clusters.conf

```diff
@@ -1,8 +1,8 @@
-# NginxK8sLB HTTP configuration, for L7 load balancing
+# NGINX Loadbalancer for K8s HTTP configuration, for L7 load balancing
```

Comment:
I think we should use the full name "Kubernetes" here, e.g.: NGINX Loadbalancer for Kubernetes